src/test/java/bio/terra/service/resourcemanagement/MetadataDataAccessUtilsTest.java

```diff
@@ -0,0 +1,95 @@
+package bio.terra.service.resourcemanagement;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyBoolean;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.when;
+
+import bio.terra.app.model.AzureCloudResource;
+import bio.terra.app.model.AzureRegion;
+import bio.terra.common.category.Unit;
+import bio.terra.model.AccessInfoAzureModel;
+import bio.terra.model.BillingProfileModel;
+import bio.terra.service.dataset.AzureStorageResource;
+import bio.terra.service.dataset.Dataset;
+import bio.terra.service.dataset.DatasetSummary;
+import bio.terra.service.dataset.DatasetTable;
+import bio.terra.service.filedata.azure.blobstore.AzureBlobStorePdao;
+import bio.terra.service.filedata.azure.util.BlobContainerClientFactory;
+import bio.terra.service.filedata.azure.util.BlobSasUrlFactory;
+import bio.terra.service.resourcemanagement.azure.AzureStorageAccountResource;
+import java.util.List;
+import java.util.UUID;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+@Category(Unit.class)
+public class MetadataDataAccessUtilsTest {
+
+  @InjectMocks private MetadataDataAccessUtils metadataDataAccessUtils;
+
+  @Mock private static ResourceService resourceService;
+
+  @Mock private static AzureBlobStorePdao azureBlobStorePdao;
+
+  @Mock private BlobContainerClientFactory blobContainerClientFactory;
+
+  @Mock private BlobSasUrlFactory blobSasUrlFactory;
+
+  private Dataset azureDataset;
+  private DatasetSummary azureDatasetSummary;
+  private BillingProfileModel defaultProfileModel;
+
+  @Before
+  public void setup() throws Exception {
+    MockitoAnnotations.openMocks(this).close();
```

Comment:
Not sure what this line does?  I think you could do away with this by adding the:
```@RunWith(MockitoJUnitRunner.class)``` annotation to the class